hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeRpcServer.java

```diff
@@ -979,7 +979,10 @@ public boolean complete(String src, String clientName,
                           ExtendedBlock last,  long fileId)
       throws IOException {
     checkNNStartup();
-    return namesystem.completeFile(src, clientName, last, fileId);
+    boolean result = namesystem.completeFile(src, clientName, last, fileId);
+    LOG.debug("complete: src={}, clientName={}, fileId={}, result={}.",
```

Comment:
The others use somethine like:
```
blockStateChangeLog.debug("*BLOCK* 
```
Should we use something similar?

Reply:
@goiri Thanks sir, I have updated it, please help me review it again. Thanks